### PR TITLE
add information about PyLadies Lunch

### PR DESCRIPTION
**Pull request by glemaitre**
add the information for pyladies lunch

### CODE REVIEW

**Comment by glemaitre** [comment]
@marenwestermann @maikia @mrastgoo are you fine with the current content? Do you want to add additional information?

File: content/pages/2022/pyladies_lunch.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Thanks to the supports of EuroSciPy organizers, our PyLadies Lunch will take
+place on Wednesday 31st of August.
+
+This is a great opportunity for women attending the conference to meet, make
+friends, and share experiences in a relaxing environment. So if you identify
+yourself as a woman, we would like to invite you to join us and make new
+friends or catch up with old friends.
+
+* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022
```
Comment (maikia) [inline content/pages/2022/pyladies_lunch.md:30]:
```suggestion
* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022. If you don't identify as a woman but want to learn more about Pyladies you are also welcome to join.
```
Just a suggestion. Feel free to discard it.
my motive: that's how I learnt about Pyladies years ago: from a guy who attended Pyladies event on one of the conferences and told me about them

**Review by maikia — APPROVED** [review]
great! thanks! one suggestion, otherwise all good

File: content/pages/2022/pyladies_lunch.md (hunk 1 above)
Comment (marenwestermann) [reply content/pages/2022/pyladies_lunch.md:30]: I'm also happy with inviting allies. There are a lot of nice people who don't identify as female who want to learn about our work and how they can best support us and I think this is a good opportunity.

**Comment by marenwestermann** [comment]
I'm in favour of @maikia's suggestion. Otherwise looks good!

**Review by marenwestermann — APPROVED** [review]

File: content/pages/2022/pyladies_lunch.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Thanks to the supports of EuroSciPy organizers, our PyLadies Lunch will take
+place on Wednesday 31st of August.
+
+This is a great opportunity for women attending the conference to meet, make
+friends, and share experiences in a relaxing environment. So if you identify
+yourself as a woman, we would like to invite you to join us and make new
+friends or catch up with old friends.
+
+* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022. If you don't identify as a woman but want to learn more about Pyladies you are also welcome to join.
```
Comment (glemaitre) [inline content/pages/2022/pyladies_lunch.md:30]:
```suggestion
* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022.
If you don't identify as a woman but want to learn more about Pyladies you are
also welcome to join.
```

**Comment by glemaitre** [comment]
Great. I accepted the changes. Merging.
Thanks.

Do not hesitate to open a PR or an issue to ask to request any change.

**Pull request merged**